docs/docs/debugging-the-build-process.md

```diff
@@ -6,6 +6,7 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
 
 In this guide you will learn how to debug some code using:
 
+- [Debugging with Node.js' built-in console](#debugging-with-nodejs-built-in-console)
```

Comment:
One more thing: this hard-coded table of contents can be removed as there's an automatic one on the page now.